charts/kafdrop/values.yaml

```diff
@@ -0,0 +1,1034 @@
+  #
+
+  ## Custom Container Name
+  # deployment.containerName -- Configure Container Name
+  # @default -- `.Chart.Name`
+  containerName:
+
+  ## Docker Image definition.
+  image:
+    # deployment.image.registry -- Configure Docker Registry. Will be overwritten if set by global variable.
+    registry: docker.io
+
+    # deployment.image.repository -- Configure Docker Repository
+    repository: obsidiandynamics/kafdrop
+
+    # deployment.image.pullPolicy -- Configure Docker Pull Policy. Will be overwritten if set by global variable.
+    pullPolicy:
+
+    # deployment.image.tag -- Configure Docker Image tag
+    # @default -- Tag defaults to `.Chart.Appversion`, if not set
+    tag: ""
+
+  ## Container Resources
+  # deployment.resources -- Configure Container [Resource](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/)
+  resources:
+    {}
+  #  requests:
+  #    memory: "64Mi"
+  #    cpu: "250m"
+  #  limits:
+  #    memory: "128Mi"
+  #    cpu: "500m"
+
+  ## Container Command
+  # deployment.command -- Configure executed container command
+  command:
+    {}
+
+  ## Container Command Args
+  # deployment.args -- Configure arguments for executed command
+  args:
+    {}
+
+  ## Environment Variables
+  # deployment.environment -- Configure Environment Variables (Refer to values.yaml)
+  environment:
+    {}
+  # - name: "MY_ENV"
+  #   value: "someValue"
+  #   secret: true
+  # - name: SOME_SPEC
+  #   valueFrom:
+  #     fieldRef:
+  #       fieldPath: spec.*
+  #
+
+  ## Container Ports
+  # deployment.ports -- Configure Container Ports
+  ports:
+    {}
+  # - name: "health"
+  #   containerPort: 9090
+  #   protocol: TCP
+
+  ## Additional Volume Mounts
+  # deployment.volumeMounts -- Configure Container [volumeMounts](https://kubernetes.io/docs/tasks/configure-pod-container/configure-volume-storage/)
+  volumeMounts:
+    {}
+  #  - name: "cache"
+  #    mountPath: /cache
+  #  - name: "share"
+  #    mountPath: /var/data/share
+
+  ## Container SecurityContext
+  # deployment.securityContext -- Container [SecurityContext](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/)
+  securityContext:
+    runAsNonRoot: false
+
+  ## ReadinessProbe Configuration
+  # deployment.readinessProbe -- Container [ReadinessProbe](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes)
+  readinessProbe:
+    httpGet:
+      path: "{{ include \"kafdrop.endpoint\" $ }}"
+      port: http
+    initialDelaySeconds: 60
+
+  ## LivenessProbe Configuration
+  # deployment.livenessProbe -- Container [LivenessProbe](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command)
+  livenessProbe:
+    httpGet:
+      path: "{{ include \"kafdrop.endpoint\" $ }}"
```

Comment:
Behind kafdrop.endpoint is `actuator/health` as far as I can tell.
Depending on what the `actuator/health` checks, it may be a bad idea to use it in the livenessProbe. Maybe `/actuator/info` is better suited to be used in livenessProbe?

Reply:
@florentinadolf Please adress this comment with a change or statement.

Reply:
under investigation.

Reply:
`actuator/health` returns {"status":"UP"} whereas  `actuator/info` returns artifact, name, time, version and group.

For a livenessProbe `actuator/health` is suitable.